src/platform/web/ui/general/Menu.js

```diff
@@ -15,33 +15,58 @@ limitations under the License.
 */
 
 import {TemplateView} from "./TemplateView.js";
+import { tag } from "./html.js";
 
 export class Menu extends TemplateView {
-    static option(label, callback) {
-        return new MenuOption(label, callback);
+    static optionWithButton(label, callback) {
```

Comment:
This is already used in the composer (and since redactions is merged also in BaseMessageView), so either keep the name, or ensure you've applied the name change everywhere.